mdal/mdal_utils.cpp

```diff
@@ -400,6 +401,40 @@ std::string MDAL::trim( const std::string &s, const std::string &delimiters )
   return ltrim( rtrim( s, delimiters ), delimiters );
 }
 
+std::string MDAL::systemFileName( const std::string &utf8FileName )
+{
+  std::string ret = utf8FileName;
+#if defined(WIN32)
+  ret = utf8ToWin32Recode( utf8FileName );
+#endif
+  return ret;
+}
+
+#if defined(WIN32)
+std::string MDAL::utf8ToWin32Recode( const std::string &utf8String )
```

Comment:
is this something else than https://github.com/lutraconsulting/MDAL/blob/fa15bfa4a4bc5c15195cead36ee181bcd93302e4/mdal/mdal_utils.cpp#L40 ? or different? Maybe we can make it common if it does the same thing?